streaming/base/storage/download.py

```diff
@@ -314,6 +314,24 @@ def download_from_azure_datalake(remote: str, local: str) -> None:
         raise
 
 
+def download_from_dbfs(remote: str, local: str) -> None:
+    """Download a file from remote Databricks File System to local.
+
+    Args:
+        remote (str): Remote path (dbfs).
+        local (str): Local path (local filesystem).
+    """
+    from databricks.sdk import WorkspaceClient
+
+    client = WorkspaceClient()
+    file_path = remote.lstrip('dbfs:')
+    dbfs_file = client.dbfs.download(file_path)
+    with open(local, 'wb') as f:
```

Comment:
We also need to download the file as `*.tmp` and then rename back to original. The reason behind this is that one thread downloads the file while other threads wait for the file to exist, if it is partially downloaded and if it is being loaded by the other thread then we will see some nasty traceback.

Reply:
Done